src/main/kotlin/no/skatteetaten/aurora/boober/utils/ImageStreamUtils.kt

```diff
@@ -0,0 +1,26 @@
+package no.skatteetaten.aurora.boober.utils
+
+import io.fabric8.openshift.api.model.ImageStream
```

Comment:
Finnes det en modell for ImageStreamImport i fabric8?

Reply:
Det ser desverre ikke slik ut, men det bør ikke være så vanskelig å lage denne som data classes hvis man vil ha det.

Reply:
Det er riktig, det er ingen ImageStreamImport.
Jeg tror at ImageStream dekker det behovet vi har, i hvert fall så langt jeg har fått sett i triggerRedeploy.